lib/fluent/log.rb

```diff
@@ -343,6 +343,7 @@ def enable_color(b = true)
   module PluginLoggerMixin
     def self.included(klass)
       klass.instance_eval {
+        desc 'Allows the user to set different levels of logging for each plugin.'
```

Comment:
It seems this is using the same wording with docs. Let me merge although I don't know who is "the" user.